City-of-Pori/pori-kada
# PORI-226 changes for place element

To test:

1. drush fra -y && drush cc all
2. Go to edit any place node and make sure that you can find "Read more" link field on the "Content" tab. Add any link to it.
3. Now go to any page and add a service to the page that includes that place that you just edited.
4. Make sure the place now has a link that says "Read more" and links to the url that you provided.
5. Try to access the place node as anonymous user and you should get access denied error.
